bins/packages/tpm/tpm.sh

```diff
@@ -0,0 +1,98 @@
+TPM_VERSION="5.2"
+TPM_CHECKSUM="0057615ef43b9322d4577fc3bde0e8d6"
+TPM_LINK="https://github.com/tpm2-software/tpm2-tools/releases/download/${TPM_VERSION}/tpm2-tools-${TPM_VERSION}.tar.gz"
+
+TSS_VERSION="3.2.0"
+TSS_CHECKSUM="0d60d0df3fd0daae66881a3022281323"
+TSS_LINK="https://github.com/tpm2-software/tpm2-tss/releases/download/${TSS_VERSION}/tpm2-tss-${TSS_VERSION}.tar.gz"
+
+download_tss() {
+    download_file $TSS_LINK $TSS_CHECKSUM
+}
+
+extract_tss() {
+    if [ ! -d "tpm2-tss-${TSS_VERSION}" ]; then
+        echo "[+] extracting: tpm2-tss-${TSS_VERSION}"
+        tar -xf ${DISTDIR}/tpm2-tss-${TSS_VERSION}.tar.gz -C ${WORKDIR}
+    fi
+}
+
+prepare_tss() {
+    echo "[+] preparing tpm-tss"
+    apt-get install libjson-c-dev
+    apt-get install libcurl4-nss-dev
+    ./configure --prefix=/usr
+}
+
+compile_tss() {
+    echo "[+] compiling tpm-tss"
+    make ${MAKEOPTS}
+}
+
+install_tss() {
+    echo "[+] installing tpm-tss"
+    make DESTDIR="${ROOTDIR}" install
+}
+
+download_tpm() {
+    download_file $TPM_LINK $TPM_CHECKSUM
+}
+
+extract_tpm() {
+    if [ ! -d "tpm2-tools-${TPM_VERSION}" ]; then
+        echo "[+] extracting: tpm2-tools-${TPM_VERSION}"
+        tar -xf ${DISTDIR}/tpm2-tools-${TPM_VERSION}.tar.gz -C ${WORKDIR}
+    fi
+}
+
+prepare_tpm() {
+    echo "[+] preparing tpm"
+    github_name "tpm-${TPM_VERSION}"
```

Comment:
I added this line (check my other comment) because it's needed to let the build system know what is the full package name (name+version) to be created on the hub. After pushing this you can already see that this package was created https://hub.grid.tf/tf-autobuilder/tpm-5.2.flist.md

If you expand the flist content you will see all the files that are included in the package. It looks good from the first look, but it also include some files that are not needed. This include all ".h" files, all the ".a" files (static libs) and all the man pages and bash-completion scripts.

> I am wondering if there is a `make install` flag that avoid copying those files and will only install the necessary binaries and libraries please check that, otherwise those files need to be deleted from the output directory before publishing